client/src/actions/index.js

```diff
@@ -20,23 +21,23 @@ export const getWorkspaces = () => {
   .catch(error => console.log(error));
 };
 
-const getDescription = (id) => {
+export const getDescription = (id) => {
```

Comment:
personally I like one export statement at bottom of file, but this totally works